restarter/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	http.HandleFunc("/restart", restartSignature)
+
+	var host, port string
+	flag.StringVar(&host, "host", "0.0.0.0", "Listen host")
+	flag.StringVar(&port, "port", "8081", "Listen port")
+	flag.Parse()
```

Comment:
This is slightly cleaner.
```suggestion
	host := flag.String(&host, "host", "0.0.0.0", "Listen host")
	port := flag.String(&port, "port", "8081", "Listen port")
	flag.Parse()
```

Reply:
Also address could be just 0.

Reply:
I made it 0 but the above suggestion is not correct as we assign to &host (first param of flag.StringVar) and if not created before flag.StringVar, it is not compiled

Reply:
I fixed the exmaple.

Reply:
done